transport-native-unix-common/pom.xml

```diff
@@ -46,6 +48,31 @@
 
   <build>
     <plugins>
+      <!-- Download the netty-jni-util source -->
+      <plugin>
+        <artifactId>maven-scm-plugin</artifactId>
+        <version>1.11.2</version>
+        <executions>
+          <execution>
+            <id>get-netty-jni-util</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>checkout</goal>
+            </goals>
+            <configuration>
+              <checkoutDirectory>${jniUtilCheckoutDir}</checkoutDirectory>
+              <connectionType>developerConnection</connectionType>
+              <developerConnectionUrl>scm:git:https://github.com/netty/netty-jni-util.git</developerConnectionUrl>
+              <!-- TODO: Change this to a release tag once we did a release -->
+              <scmVersion>main</scmVersion>
```

Comment:
Before we merge this one we should tag a version for netty-jni-util and use it. I just wanted to verify first that it works for netty and netty-tcnative